### ISSUE TITLE
Communication error , then crash

### ISSUE THREAD

**Issue by overflywingo** [F4stFr3ddy/homebridge-jablotron-alarm#16]
Hello, 
First at all thanks for having created that plugin :)   but I have an issue with it 
I'm using the latest version of the plugin 1.1.0
I hope you can help me ? 

1) Output of the config helper 

Service_id: 44355 
Service_name: ALARME xxxxx
Alarm_type: JA100

This alarm has the following segments: 

Segment_id: STATE_1
Segment_key: section_1
Keyboard_key: keyboard_8_2
Segment_name: Tout

2) My config.json (sensitive data masked with a "x"

 "accessory": "Homebridge-Jablotron",
"name": "ALARME xxxxx",
 "username": "xxxxxxxxxx@gmail.com",
"password": "xxxxxxxxxx.",
"pincode": "xxxx",
 "service_id": "xxxxx",
"segment_id": "STATE_1",
 "segment_key": "section_1",
"keyboard_key": " keyboard_8_2"

3) Home bridge console log 

[2019-2-8 13:17:21] Loaded plugin: homebridge-jablotron
[2019-2-8 13:17:21] Registering accessory 'homebridge-jablotron.Homebridge-Jablotron'
..
[2019-2-8 13:17:31] [ALARME xxxxx] Initializing Homebridge-Jablotron accessory...
...
then I launch HOME APP on my iPhone 
...
[2019-2-8 13:20:08] [ALARME xxxxx] Getting state of alarm ...
[2019-2-8 13:20:08] [ALARME xxxxx] Using cached serviceId.
[2019-2-8 13:20:09] [ALARME xxxxx] Obtaining sessionId.
[2019-2-8 13:20:09] [ALARME xxxxx] Getting state of alarm ...
[2019-2-8 13:20:09] [ALARME xxxxx] Using cached serviceId.
[2019-2-8 13:20:09] [ALARME xxxxx] Obtaining sessionId.
[2019-2-8 13:20:20] [ALARME xxxxx] Alarm state: unset
[2019-2-8 13:20:22] [ALARME xxxxx] Alarm state: unset
....
Up to here everything looks fine, the alarm status (unset) is well detected and correctly displayed in home app. 
When I try to activate the alarm in home app (long press and select for instance 'night') then I got this 
...
[2019-2-8 13:25:54] [ALARME xxxxx] Setting alarm state: 2
[2019-2-8 13:25:54] [ALARME xxxxx] Using cached serviceId.
[2019-2-8 13:25:54] [ALARME xxxxx] Switching section section_1 (using  keyboard_8_2) to new state: partialSet
[2019-2-8 13:25:54] [ALARME xxxxx] Using cached sessionId.
[2019-2-8 13:26:13] [ALARME xxxxx] { status: false,
  error_status: 'error_dialog',
  error_type: 'dialog',
  error_message: 'Communication error',
  checksum: '62bf2ec18642c87aa75f55b82c40426e',
  server_id: '2409322-1549628773',
  client_id: null }
[2019-2-8 13:26:13] [ALARME xxxxx] Cannot handle error:
[2019-2-8 13:26:13] [ALARME xxxxx] Error: error_dialog
    at /usr/local/lib/node_modules/homebridge-jablotron/Jablotron/jablotronClient.js:24:31
    at IncomingMessage.resp.on (/usr/local/lib/node_modules/homebridge-jablotron/Jablotron/jablotronClient.js:53:17)
    at IncomingMessage.emit (events.js:185:15)
    at endReadableNT (_stream_readable.js:1101:12)
    at process._tickCallback (internal/process/next_tick.js:114:19)

**Comment by misncz** [comment]
That looks like communication error between Jabltron server and your unit. Are you sure Jabltron account credentials together with configured PIN code are allowed to arm/disarm your unit? You can fine tune access privilegies at jablotron.net

**Comment by F4stFr3ddy** [comment]
I would expect to see `error_message: 'Operation failed'` if it was permission related. However its worth to check your credentials and if your account is authorized to (dis)arm the system.

Also, please remove your service_id from the comment as it might pose a security risk if people know where the alarm is installed.

**Comment by overflywingo** [comment]
Hi, indeed it looks like that I've a problem trying to activate the alarm from both the web interface (www.jablonet.net) as well as from the IOS app.  So I've to fix this first.  So some homework.   In the meantime I  closed this issue. 

PS : service id cleaned.

**Comment by overflywingo** [comment]
Hi, to fix my local issue with my Jablotron, I've been forced to reboot the JA-100 alarm system.  And after I was able to arm the system w/o any problem from the Jablotron web interface and Jablotron app.

Then I checked again with the plugin ... but bad luck ... still not OK.   
Looking again all the plugin parameters I realized that by mistake I had an extra space left here : "keyboard_key": " keyboard_8_2" (just before keyboard_8_2) .. 
I removed it ... and voilà !  Everthing OK  :)    I'm happy.
